### DISCUSSION TITLE
`package 'slam_toolbox' not found` on Pi5 through Docker

### DISCUSSION THREAD

**Discussion by bdawkinsroj** [iRobotEducation/create3_docs#599] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

latest humble release

### Which RMW is your robot running?

I don't know

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

I'm able to copy the workspace and build it in the docker container, however when I go to start using the lidar I can run the sensors_launch.py but not the slam_toolbox_launch.py. It says that it isn't found even though its in the same place as sensors_launch.py. Anyone know how to fix this?

**Comment by brianabouchard** [comment]
Hello! I haven't tested this in a docker container. However, what you're describing sounds like you may not have run the source command in your second window where you are running the slam toolbox launch script. Did you run `source ~/create3_examples_ws/install/local_setup.sh ` before trying to run the slam toolbox launch script?

**Reply by bdawkinsroj** [reply]
I did run the source prior to attempting slam. I get the error: 
[ERROR] [launch]: Caught exception in launch (see debug for traceback): "package 'slam_toolbox' not found, searching: ['/create3_ws/install/create3_teleop', '/create3_ws/install/create3_republisher', '/create3_ws/install/create3_lidar_slam', '/create3_ws/install/create3_examples_py', '/create3_ws/install/create3_coverage', '/create3_ws/install/create3_examples_msgs', '/opt/ros/humble']"

**Reply by brianabouchard** [reply, marked as answer]
Hmm, that would seem to indicate the package has not been installed. In your container, run the following commands:

`cd create3_examples_ws`
`rosdep install --from-path src --ignore-src -yi`

Does that trigger an install?

**Reply by bdawkinsroj** [reply]
That fixed it, thank you so much!
